### DISCUSSION TITLE
[solved] Does EnOcean PTM 216Z work with a SONOFF EFR32MG21

### DISCUSSION THREAD

**Discussion by Friesoch** [Koenkk/zigbee2mqtt#15807] [Devices]
Hello Everybody

I used to have a CC2530 where my four PTM 216Z worked fine. I now replaced my old stick with a Sonoff Zigbee 3.0 USB Dongle Plus model ZBDongle-E (Based EFR32MG21). I got all the classic Zigbee Devices working fine except the [EnOcean PTM 216 Z](https://www.zigbee2mqtt.io/devices/PTM_216Z.html). Do these devices needed to be "unpaired" first (and if yes how?)? Does the Sonoff support EnOcean Devices? 

Thanks for any help!
KR
Frieso

**Comment by Kostecki** [comment]
I would like to know this as well 👍

**Reply by Friesoch** [reply]
@Koenkk do you know if it works?

**Reply by Koenkk** [reply]
I don't know, @kirovilya do you know if these adapters support green power devices?

**Reply by kirovilya** [reply]
should work, you need to look at herdsman logs

**Reply by Friesoch** [reply]
@kirovilya I started zigbee2mqtt with herdsman in debug mode ( xxx:/opt/zigbee2mqtt $ DEBUG=zigbee-herdsman* npm start ) but I do not see any message when trying to pair.

When I push a button on my PTM 216Z I see, after about 20 Seconds the following message in the Zigbee2mqtt Log-File:

`Received Zigbee message from 'Coordinator', type 'commandNotification', cluster 'greenPower', data '{"data":[25,91,2,11,254,0],"type":"Buffer"}' from endpoint 242 with groupID null, ignoring since it is from coordinator
`

As I replaced my Zigbee-USB-Stick from CC2530 to the new Sonoff, could this be the problem? Does the PTM 216Z or the Hue-Bulp remember the old stick ID?

**Reply by Kostecki** [reply]
Huh... I just ran Zigbee2MQTT with the debug zigbee-herdsman to try and post some logs as well and it just worked. I could pair one of my Green Power devices no problem. I disabled the debugging, tried again and pairing still worked without problems.

I used the exact same procedure of pairing as i have done multiple times in the past, so i have absolutely no idea about why it suddenly just worked, but i'll take it.

**Reply by Friesoch** [reply]
@Kostecki It does work :) As I migrated from an older Zigbee Stick to Sonoff I first had to factory-reset the PTM 216Z by pushing all 4 Buttons for 10 Seconds. Then I was able to pair the PTM 216Z with my new Sonoff ERF32MG21- the PTM 216Z it has to be next to a i.e. Philips Hue Bulp to pair it again as written in the https://www.zigbee2mqtt.io/devices/PTM_215Z.html manual.
